oe_theme.theme

```diff
@@ -1045,3 +1045,41 @@ function oe_theme_preprocess_language_selection_page_content(&$variables) {
     $language['path'] = $language['href'];
   }
 }
+
+/**
+ * Implements template_preprocess_block__oe_theme_main_page_content().
+ */
+function oe_theme_preprocess_block__oe_theme_main_page_content(&$variables) {
+  $variables['attributes']['class'][] = 'ecl-u-mv-l';
+}
+
+/**
+ * Implements hook_preprocess_table().
+ */
+function oe_theme_preprocess_table(array &$variables): void {
+  $variables['zebra'] = 'true';
+
+  $name_col = [];
+  foreach ($variables['header'] as $key => &$cell) {
```

Comment:
Defining the $key variable is not required.

Reply:
done.